ios/SailthruMobile.framework/Headers/STMNotificationCategory.h

```diff
@@ -6,7 +6,7 @@
 //  Copyright © 2019 Sailthru. All rights reserved.
 //
 
-#import "CarnivalNotificationCategory.h"
+#import "STMNotificationCategory.h"
```

Comment:
Just checking, am I right in thinking these types of files are just direct copies (replacing `Carnival` with `STM`) of the deleted ones with `Carnival` in the name?

Reply:
Exactly, yeah, the STM classes are the rebranded versions of the same classes. This is just updating the SailthruMobile.framework with the latest version, which not longer contains references to the old `Carnival` classes at all.